history-of-the-quakers/original/06-book-1-chapter-3.adoc

```diff
@@ -67,13 +76,16 @@ or until they shall find sufficient Security to be of good Behaviour,
 or be thence delivered by Order from ourselves.
 Hereof you are not to fail.
 
+[.signed-section-closing]
 Given under our Hands and Seals this 30th Day of October, 1650,
 
+[.signed-section-signature]
 Ger.
-Bennet,
+Bennet
 
+[.signed-section-signature]
 Nath.
-Barton.`"
+Barton
 
```

Comment:
I also noticed I didn't end the embedding either. :/  The linter didn't catch it though on the PR I don't think. I thought after you sent the PR up you could fix things the linter caught? Maybe I'm not remembering right.

Reply:
Yeah, that's weird -- I pulled down a copy of the PR and the linter caught it for me on my computer.  When your making the other changes, just add a `--` here after Nath. Barton and it should be good.